### ISSUE TITLE
pprof: Profile.Add of duplicate value

### ISSUE THREAD

**Issue by ricoyoung345** [hybridgroup/gocv#865]
```text
lab.vsearch.goods.run.log.2021-06-28:panic: pprof: Profile.Add of duplicate value
lab.vsearch.goods.run.log.2021-06-28-
lab.vsearch.goods.run.log.2021-06-28-goroutine 22068 [running]:
lab.vsearch.goods.run.log.2021-06-28-runtime/pprof.(*Profile).Add(0xc000557aa0, 0x118e560, 0x7fa1542d9790, 0x1)
lab.vsearch.goods.run.log.2021-06-28-   /usr/local/go/src/runtime/pprof/pprof.go:298 +0x295
lab.vsearch.goods.run.log.2021-06-28-gocv.io/x/gocv.newMat(...)
lab.vsearch.goods.run.log.2021-06-28-   /opt/tiger/compile_path/pkg/mod/gocv.io/x/gocv@v0.27.0/mat_profile.go:70
lab.vsearch.goods.run.log.2021-06-28-gocv.io/x/gocv.(*Mat).Region(0xc000d44520, 0x3d, 0x18a, 0x83, 0x256, 0x7fa100000220, 0x0, 0x0, 0x7fa13c019750)
lab.vsearch.goods.run.log.2021-06-28-   /opt/tiger/compile_path/pkg/mod/gocv.io/x/gocv@v0.27.0/core.go:564 +0x8e
lab.vsearch.goods.run.log.2021-06-28-code.byted.org/lab/vsearch/go/utils/image_cv.(*ImageCV).Crop(0xc000d44520, 0x3d, 0x18a, 0x83, 0x256, 0xc005a58880)
lab.vsearch.goods.run.log.2021-06-28-   /opt/tiger/compile_path/src/code.byted.org/lab/vsearch/go/utils/image_cv/image_cv.go:79 +0x87
lab.vsearch.goods.run.log.2021-06-28-code.byted.org/lab/vsearch/go/services/goods/tools.LoadCrops.func3(0x0, 0xc000d44520, 0xc003eace60, 0xc005161b00)
lab.vsearch.goods.run.log.2021-06-28-   /opt/tiger/compile_path/src/code.byted.org/lab/vsearch/go/services/goods/tools/concurrent.go:131 +0x813
lab.vsearch.goods.run.log.2021-06-28-created by code.byted.org/lab/vsearch/go/services/goods/tools.LoadCrops.func4
lab.vsearch.goods.run.log.2021-06-28-   /opt/tiger/compile_path/src/code.byted.org/lab/vsearch/go/services/goods/tools/concurrent.go:196 +0x193
```

## Steps to Reproduce

run the following code concurrently
![image](https://user-images.githubusercontent.com/11331995/123553167-3b0d3480-d7ac-11eb-9cec-11d31df277d4.png)
```golang
func (icv *ImageCV) Crop(left, top, right, bottom int) *ImageCV {
	croppedMat := icv.mat.Region(image.Rect(left, top, right, bottom))
	// defer func() {
	// 	_ = croppedMat.Close()
	// }()
	// resultMat := croppedMat.Clone()
	// return &ImageCV{mat: resultMat}
	return &ImageCV{mat: croppedMat}
}
```
1. 
2.
3.
4.

## Your Environment

* Operating System and version: debian 8.0
* OpenCV version used: 4.5.2
* How did you install OpenCV? use [Dockerfile.opencv](https://github.com/hybridgroup/gocv/blob/release/Dockerfile.opencv)
* GoCV version used: 0.27.0
* Go version: 1.16
* Did you run the `env.sh` or `env.cmd` script before trying to `go run` or `go build`?

**Comment by deadprogram** [comment]
Hello @ricoyoung345 the current `dev` branch should have a fix for that https://github.com/hybridgroup/gocv/commit/4c86e5f770034771df5b4fca62663fffdeda8027

If you are able to try it out, please let me know if it works for you.

**Comment by ricoyoung345** [comment]
> Hello @ricoyoung345 the current `dev` branch should have a fix for that [4c86e5f](https://github.com/hybridgroup/gocv/commit/4c86e5f770034771df5b4fca62663fffdeda8027)
> 
> If you are able to try it out, please let me know if it works for you.

problem solved on branch dev
